src/cache.ts

```diff
@@ -208,6 +210,13 @@ function isSameObject<T extends KubernetesObject>(o1: T, o2: T): boolean {
     return o1.metadata!.name === o2.metadata!.name && o1.metadata!.namespace === o2.metadata!.namespace;
 }
 
+function isSameVersion<T extends KubernetesObject>(o1: T, o2: T): boolean {
+    return (
+        o1.metadata!.resourceVersion !== undefined &&
```

Comment:
probably worth testing for null also?

Reply:
thanks for the review! I will add the null check.